docs/modules/glob.md

```diff
@@ -0,0 +1,37 @@
+# Globbing
+
+Most globbing libraries are bloated and have lighter and faster alternatives.

```

Comment:
nit: i would reword throughout as `glob libraries`

also this should probably say `Many glob libraries` rather than `Most` (since we will be reducing it over time, so it won't be true anymore)